tox.ini

```
@@ -3,7 +3,8 @@ envlist = py27,py35,py36,py37
 skip_missing_interpreters = True
 
 [testenv]
-extras =
+deps = -r {toxinidir}/requirements.txt
```

Doesn’t this work just the same?

```suggestion
deps = -r requirements.txt
```